src/tests/ftest/scrubber/aggregation.yaml

```diff
@@ -39,20 +39,16 @@ pool:
 container:
   type: POSIX
   control_method: daos
-  oclass: RP_2GX
   properties: "cksum:sha512,rd_fac:1"
 ior:
   ior_timeout: 60
-  client_processes:
-    np: 6
   test_file: daos:testFile
   dfs_destroy: false
   flags: "-v -W -w -r -R -k"
   api: DFS
-  transfer_size: 1M
-  block_size: 8G
-  dfs_oclass: RP_2G1
-  dfs_dir_oclass: RP_2GX
+  transfer_size: 5M
```

Comment:
Why a 5M transfer size? It might not be bad, but I'm curious how 5M was chosen

Reply:
It seems the first ior is quick: `run_ior_with_pool`.
But then then `run_ior_and_check_scruber_status` takes a few minutes.

Also, it seems that IOR run fails?
https://build.hpdd.intel.com/job/daos-stack/job/daos/job/PR-13766/7/artifact/Functional%20Hardware%20Medium/scrubber/aggregation.py/job.log
```
2024-03-02 14:37:09,598 process          L0416 DEBUG| [stdout] ERROR: dfs_read(0x7f5651878000, 1048576) failed (5): Input/output error
2024-03-02 14:37:09,598 process          L0416 DEBUG| [stdout] , (aiori-DFS.c:806)
2024-03-02 14:37:09,598 process          L0416 DEBUG| [stderr] application called MPI_Abort(MPI_COMM_WORLD, -1) - process 0

```

Reply:
I didn't have a particular transfer size in mind - just that it tends to either be 1M or a multiple of the redundancy, or data/parity in some way if EC. 5M is fine here

Reply:
IOR failure is expected... The faults are slowly injected and IOR will fail since scrubber is working the background and ejecting the targets. 

First IOR should be fast... Second IOR will take time...

Reply:
Oh, right :)